indico/web/templates/forms/_form.html

```diff
@@ -9,13 +9,14 @@
 {%- endmacro %}
 
 {% macro form_header(form, method='post', action='', orientation='horizontal', id=none, classes='', multipart=false,
-                     i_form=true, no_ajax=false) -%}
+                     i_form=true, no_ajax=false, ajax_form=false) -%}
```

Comment:
Sounds like a very good reason for me to keep this one in the caller too instead of adding it to the normal args